tests/client/client_common.py

```diff
@@ -120,6 +171,45 @@ def test_push_model_project_contains_two_tasks(ebnt: Ebonite, model: Model):
     assert project.tasks.get(task.id) == task1
 
 
+def delete_model_ok(ebnt: Ebonite):
+    task = ebnt.get_or_create_task('Project', 'Task')
+    model = Model(name='Model', task_id=task.id)
+    model = ebnt.meta_repo.create_model(model)
+    ebnt.delete_model(model)
+
+    assert ebnt.meta_repo.get_model_by_id(model.id) is None
+
+
+def delete_model_cascade_ok(ebnt: Ebonite, model: Model):
+    task = ebnt.get_or_create_task('Project', 'Task')
+    model = ebnt.push_model(model, task)
+    image = Image(model_id=model.id, name='Image')
+    image = ebnt.meta_repo.create_image(image)
+    ebnt.delete_model(model, cascade=True)
+
+    assert ebnt.meta_repo.get_image_by_id(image.id) is None
```

Comment:
For such checks I highly recommend checking negation of check before operation. E.g., in this test check that image with such id actually exists before call to `delete_model`. This ensures that something happened during the call.

Reply:
The same holds for all tests, not just this one.

Reply:
I propose to make this more explicit:
`assert ebnt.meta_repo.get_image_by_id(image.id) is not None`
instead of
`assert image is not None`.

It should be clear that some condition doesn't hold before calling and holds after.